packages/utils/src/namespace.ts

```diff
@@ -109,8 +108,7 @@ export const ConnectArgumentsStruct = object({
 });
 export type ConnectArguments = Infer<typeof ConnectArgumentsStruct>;
 
-// TODO: Fix this type.
-export const RequestArgumentsStruct: any = assign(
+export const RequestArgumentsStruct = assign(
```

Comment:
How did this get fixed?

Reply:
Not sure, I removed `any` and it works. 🤷